src/interfaces.ts

```diff
@@ -10,3 +10,6 @@ export interface LoginInput {
   password: string;
   rememberMe?: boolean;
 }
+export interface queryUserInput {
```

Comment:
```suggestion
export interface UserQueryInput {
```

Interfaces no geral a gente nomeia como `PascalCase`